products/controlm.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Control-M
+category: server-app
+iconSlug: bmcsoftware
+permalink: /controlm
+alternate_urls:
+-   /bmccontrolm
+-   /ctrlm
+versionCommand: ctm -v
+releasePolicyLink: http://www.bmc.com/support/product-support-policy.html
+eolColumn: Limited Support
+eoasColumn: Full Support
+
+identifiers:
+-   cpe:2.3:a:bmc:control-m
+
+# Latest releases can be found on https://docs.bmc.com/docs/controlm/.
+releases:
+-   releaseCycle: "9.0.21"
+    releaseDate: 2022-09-08
+    eoas: 2025-09-08
+    eol: 2027-09-08
+    latest: "9.0.21.200"
+    latestReleaseDate: 2024-01-08
+    link: https://docs.bmc.com/docs/controlm/90201/control-m-9-0-21-200-release-notes-1238266316.html
+
+-   releaseCycle: "9.0.20"
+    releaseDate: 2020-07-24
+    eoas: 2024-04-30
+    eol: 2025-07-23
+    latest: "9.0.20.200"
+    latestReleaseDate: 2021-10-28
+    link: https://docs.bmc.com/docs/controlm/9020/control-m-version-9-0-20-200-release-notes-1007971086.html
+
+-   releaseCycle: "9.0.19"
+    releaseDate: 2019-03-04
+    eoas: 2022-12-31
+    eol: 2024-03-04
+    latest: "9.0.19.200"
+    latestReleaseDate: 2020-01-16
+    link: https://docs.bmc.com/docs/controlm/9019/control-m-version-9-0-19-200-release-notes-862843904.html 
+
+-   releaseCycle: "9.0.18"
+    releaseDate: 2018-01-03
+    eoas: 2021-03-31
+    eol: 2023-03-31
+    latest: "9.0.18.200"
+    latestReleaseDate: 2018-10-04
+    link: https://docs.bmc.com/docs/controlm/9018/control-m-version-9-0-18-200-release-notes-830144784.html
+
+-   releaseCycle: "9.0.0"
+    releaseDate: 2015-07-15
+    eoas: 2018-08-01
+    eol: 2020-10-31
+    latest: "9.0.00.200"
+    latestReleaseDate: 2016-06-21
+    link: https://documents.bmc.com/supportu/952/78/79/477879/477879.pdf
+
+---
+
+
+> [Control-M](https://www.bmc.com/it-solutions/control-m.html) is an enterprise workload automation solution that helps organizations manage and orchestrate their business processes, applications, and data workflows.
+
+BMC provides at least three years of full support from the release date of each release, with a further two years of limited support for high impact problems.
```

Comment:
```suggestion
{: .warning }
> This page is using the dates from [Control-M/Server for UNIX and Microsoft Windows](https://webapps.bmc.com/support/faces/az/prodallversions.jsp?seqid=122988),
> which are largely shared by the other Control-M components on [the Supported Product List page](https://webapps.bmc.com/support/faces/az/supportlisting.jsp).

BMC provides at least three years of full support from the release date of each release, with a further two years of limited support for high impact problems.
```

To address https://github.com/endoflife-date/endoflife.date/pull/5118#discussion_r1707577213. WDYT @baggers27 ?

Reply:
Looks great, thanks.